<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>6820</width>
    <height>12120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="trainer_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1000</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
color: rgb(255, 255, 255);
font: 18pt &quot;HY견고딕&quot;;</string>
   </property>
   <property name="text">
    <string>TRAINER K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="weight_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>960</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(img/weight_btn_4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>891</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 91, 91);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>320</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
font: 22pt &quot;HY견고딕&quot;;</string>
   </property>
   <property name="text">
    <string>하체 전체 운동</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
font: 16pt &quot;HY견고딕&quot;;</string>
   </property>
   <property name="text">
    <string>(앞뒤 허벅지, 엉덩이)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>1140</y>
     <width>1081</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>다리의 위치를 맞춰주세요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="tip_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>1700</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/info_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>1700</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/back_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>1700</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/home_btn.png);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Virtual Fitness</string>
   </property>
  </widget>
  <widget class="QLabel" name="datetime_lb">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>날짜, 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1091</width>
     <height>1611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1081</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(img/plan_3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY견고딕</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius: 15px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>logout_btn</zorder>
  <zorder>trainer_btn</zorder>
  <zorder>weight_btn</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>tip_btn</zorder>
  <zorder>back_btn</zorder>
  <zorder>home_btn</zorder>
  <zorder>label_5</zorder>
  <zorder>datetime_lb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
